File: examples/5.blocks-and-whitespace/for.py
from random import randint

#
# Python's syntax for loops is **much** simpler and more powerful.
#

people = [
    {"name": "Kalle", "salt": 856412},
    {"name": "Pierre", "salt": 215863}
]

# Python operates using references rather than copies of data:
for person in people:
    person["salt"] = randint(0, 999999)

File: examples/basic/5.blocks-and-whitespace/for.php
<?php

    /*
     *
     * This example is stolen right from the PHP website.
     *
     */

    $people = array(
        array('name' => 'Kalle', 'salt' => 856412),
        array('name' => 'Pierre', 'salt' => 215863)
    );

    for($i = 0; $i < count($people); ++$i) {
        $people[$i]['salt'] = mt_rand(000000, 999999);
    }

?>
